kubernetes/operationcode_backend/deployment.yml

```diff
@@ -27,37 +27,52 @@ spec:
             secretKeyRef:
               name: backend-secrets
               key: airtable_api_key
+        - name: SLACK_LEGACY_ADMIN_TOKEN 
+          valueFrom:
+            secretKeyRef:
+              name: backend-secrets
+              key: slack_legacy_admin_token 
         - name: AIRTABLE_BASE_ID
           valueFrom:
             secretKeyRef:
               name: backend-secrets
               key: airtable_base_id
+              optional: true
```

Comment:
What's this `optional: true` doing here? It's not mentioned in the PR. Also for each following reference.

Reply:
We aren't using airtable in staging, Without this environment variable set in the config file kubernetes gets mad. 

Currently airtable is used as a spreadsheet database for mentor requests.